src/crypto/CHIPCryptoPALOpenSSL.cpp

```diff
@@ -1,6 +1,7 @@
 /*
  *
  *    Copyright (c) 2020-2023 Project CHIP Authors
+ *    Copyright 2025 NXP
```

Comment:
Why this change? Generally everything is chip authors. If we add every company that contributes and years, this will grow a lot on all files.